src/js/components/organisms/colors/data/colors.data.ts

```diff
@@ -0,0 +1,46 @@
+export const data = {
+    default: {
+        title: 'Lorem ipsum .. enzo nog veel meer',
+        subtitle: 'moest er in van Martin',
+        colorList: [
+            {
+                rgbHex: '#ff1493',
+                exampleImage: {
+                    alt: 'Ultimaker Tough PLA print',
+                    mimeType: 'image/jpeg',
+                    type: 'Image',
```

Comment:
think the data should be using "real" data such as:

* https://ultimaker.com/img/materials/samples/PLA/Ultimaker-PLA-Blue@2x.png
* https://ultimaker.com/img/materials/samples/PLA/Ultimaker-PLA-Magenta@2x.png

Reply:
Let discuss this globally, images on ultimaker.com do not resize with the c-image component. The svg-mocks do. Then the question arises where to host the images and how to work without a working internet connection on your local development environment.